### PR TITLE
Added a Null Check to Fix Issue 23

### PR DESCRIPTION
**Pull request by adobito**
Added a NullCheck to fix issue 23. My last change of changing to phone was undone so it shouldn't be an issue, but I should've created my branch from a previous commit. Oops. Still, this fixes just issue 23.

### CODE REVIEW

File: user-lock-manager.smartapp.groovy [hunk 1]
```diff
@@ -115,7 +115,7 @@ def userPage(params) {
           title = "Code (Must be ${lock.latestValue('pinLength')} digits)"
         }
       }
-      input(name: "userCode${i}", type: "text", title: title, required: false, defaultValue: settings."userCode${i}", refreshAfterSelection: true)
+      input(name: "userCode${i}", type: "phone", title: title, required: false, defaultValue: settings."userCode${i}", refreshAfterSelection: true)
```
Comment (ethayer) [inline user-lock-manager.smartapp.groovy:118]: Type: 'phone' doesn't work for iOS devices.

**Comment by ethayer** [comment]
I'd accept this in, but I can't change the input type.  iOS doesn't play well with it.

**Comment by adobito** [comment]
Ill try and see if I have time to make a proper pull request on the actual
issue. I messed up my repo accidentally.

**Comment by ethayer** [comment]
I threw it into a commit to get it out quick.
https://github.com/ethayer/user-lock-manager/commit/7a02594047d9182cb444a2a8c6cb8f9c8c633cd7

if there's any other bugs you're looking at squishing, forward them over!  I'll merge them in.

**Pull request closed without merging**